components/ws-notification/index.js

```diff
@@ -0,0 +1,110 @@
+import { React, Component } from '../imports';
+import './ws-notification.scss';
+
+//const NOTIFICATIONS_SHOW_LIMIT = 3;
+const DEFAULT_NOTIFICATION_LIFETIME = 5000;
+const DEFAULT_NOTIFICATION_TYPE = 'info';
+
+export default class WSNotification extends Component {
+  constructor() {
+    super();
+    this.state = {
+      notifications: [],
+      timeoutId: null
+    };
+    // Bind functions to this scope
+    this.addNotify = this.addNotify.bind(this);
+    this.closeAllEvent = this.closeAllEvent.bind(this);
+  }
+  componentDidMount() {
+    window.addEventListener('ws-notification-open', this.addNotify);
+    window.addEventListener('ws-notification-close-all', this.closeAllEvent);
```

Comment:
I really like this approach since the application doesn't have to care if the notification component works or not.